gm4-polyfill.js

```diff
@@ -68,6 +68,24 @@ if (typeof GM_registerMenuCommand == 'undefined') {
 }
 
 
+if (typeof GM_getResourceText == 'undefined') {
+  this.GM_getResourceText = (aRes) => {
+    'use strict';
+    return GM.getResourceUrl(aRes)
+      .then(function(url) {
```

Comment:
This can/should be just

    .then(url => fetch(url))

Same for the text entry.

Reply:
Done. My javascript fu is old, so I got to learn something about arrow functions. Thanks.